go/Makefile.debian9

```diff
@@ -1,4 +1,4 @@
-IMAGES         := base main darwin arm armhf npcap
+IMAGES         := base main darwin armhf npcap
```

Comment:
Do we want to do the same thing for `armhf` in a follow-up?

Reply:
Good question, I am not sure if qemu will need to be used under the hood to produce armhf binaries when running on an arm64 worker -- given that ultimately we want to avoid any involvement of qemu when running packaging jobs on arm64 workers, I felt it's safe to keep doing this on the x86_64 worker instead.